themes/bootstrap3/templates/record/cart-buttons.phtml

```diff
@@ -3,10 +3,12 @@
   <?php $cartId = $this->source . '|' . $this->id; ?>
   <span class="btn-bookbag-toggle" data-cart-id="<?=$this->escapeHtmlAttr($this->id)?>" data-cart-source="<?=$this->escapeHtmlAttr($this->source)?>">
     <a href="#" class="cart-add hidden<?php if(!$cart->contains($cartId)): ?> correct<?php endif ?>">
-      <span class="cart-link-label btn-type-add"><?=$this->transEsc('Add to Book Bag') ?></span>
+      <i class="cart-link-icon fa fa-fw fa-plus-circle" aria-hidden="true"></i>
+      <span class="cart-link-label"><?=$this->transEsc('Add to Book Bag') ?></span>
```

Comment:
I believe that this template is shared between the results list and the record view, though I could be mistaken. Regardless of whether that detail is correct, I am seeing a problem on the record view when the book bag is enabled: when you mouse over buttons, the link underline disappears for every button except the cart button, where the underline either stays in place or simply changes its width (depending on which theme you are using).

Reply:
Taken care of! 👍